ice/src/agent/agent_internal.rs

```diff
@@ -1085,14 +1079,14 @@ impl AgentInternal {
                 tokio::select! {
                     opt_state = chan_state_rx.recv() => {
                         if let Some(s) = opt_state {
-                            let mut on_connection_state_change_hdlr = ai.on_connection_state_change_hdlr.lock().await;
-                            if let Some(f) = &mut *on_connection_state_change_hdlr{
+                            if let Some(hndlr) = &*ai.on_connection_state_change_hdlr.load() {
```

Comment:
```suggestion
                            if let Some(handler) = &*ai.on_connection_state_change_hdlr.load() {
```

let's maybe give this a more human readable name

Reply:
Done [87fca0e](https://github.com/webrtc-rs/webrtc/pull/338/commits/87fca0e4031fde7b8a9cecf37e1878c2dc2096f1)